backend/app/api/src/helpers/AdminPermissionChecker.ts

```diff
@@ -1245,6 +1136,10 @@ export class AdminPermissionChecker {
         return !!this.platformPermissions && !!this.platformPermissions.hasAccessRight(AccessRight.PlatformLoginAs);
     }
 
+    canAccess(accessRight: AccessRight): boolean {
```

Comment:
Deze naam is erg onduidelijk en maakt niet duidelijk wat de functie doet, namelijk enkel de platform permissions checken.

```suggestion
    hasPlatformAccessRight(accessRight: AccessRight): boolean {
```